x-pack/platform/plugins/shared/fleet/server/saved_objects/index.ts

```diff
@@ -1297,4 +1308,10 @@ export function registerEncryptedSavedObjects(
     // enforceRandomId allows to create an SO with an arbitrary id
     enforceRandomId: false,
   });
+  encryptedSavedObjects.registerType({
+    type: DOWNLOAD_SOURCE_SAVED_OBJECT_TYPE,
+    attributesToEncrypt: new Set([{ key: 'ssl', dangerouslyExposeValue: true }]),
+    // enforceRandomId allows to create an SO with an arbitrary id
+    enforceRandomId: false,
```

Comment:
Explanation of the need for this parameter: https://github.com/elastic/kibana/pull/208091/files#r1948774730